frontend/frontend/src/screens/HomeScreen.js

```diff
@@ -0,0 +1,69 @@
+import React, { useEffect, useState } from 'react'
+import { Form, Button, Container, Row, Col } from 'react-bootstrap'
+import { Card } from 'react-bootstrap'
+import { Link } from 'react-router-dom'
+
+import axios from 'axios'
+import Message from '../components/Message'
+import Loader from '../components/Loader'
+
+function HomeScreen() {
+    useEffect(() => {}, [])
```

Comment:
Could you describe what this hooks call is for?

Reply:
For nothing actually, this a typo that will be fixed